sandbox-backend/src/main/java/com/exadel/sandbox/team5/util/SearchCriteria.java

```diff
@@ -0,0 +1,55 @@
+package com.exadel.sandbox.team5.util;
+
+import org.springframework.data.domain.PageRequest;
+
+import java.util.Date;
+import java.util.List;
+
+public class SearchCriteria {
+    private PageRequest pageRequest;
+    private Date date;
```

Comment:
Why do we need date as an additional field?

Reply:
Got it.